### PR TITLE
Updated introduction to Corda Network, with some content removed

### PR DESCRIPTION
**Pull request by jamescarlyle**
Updated introduction to Corda Network, with some content removed now it's
available on https://corda.network.

Signed-off-by: jamescarlyle <jwgcarlyle@gmail.com>

### CODE REVIEW

File: docs/source/corda-network/index.md [hunk 1]
```diff
@@ -3,57 +3,59 @@ Corda Network
 
 Introduction to Corda Network
 -----------------------------
-Corda Network consists of Corda nodes operated by network participants, in which business transactions are created and 
-validated via Corda Distributed Applications (CorDapps) running on these nodes. Each node is identified by means of a 
-certificate issued by the Network's Certificate Authority, and will also be identifiable on a network map. 
+[Corda Network](https://corda.network/) is an publicly-available internet of Corda nodes operated by network participants. Each
```
Comment (gendal) [inline docs/source/corda-network/index.md]: is a publicly-available

File: docs/source/corda-network/index.md [hunk 2]
```diff
@@ -3,57 +3,59 @@ Corda Network
 
 Introduction to Corda Network
 -----------------------------
-Corda Network consists of Corda nodes operated by network participants, in which business transactions are created and 
-validated via Corda Distributed Applications (CorDapps) running on these nodes. Each node is identified by means of a 
-certificate issued by the Network's Certificate Authority, and will also be identifiable on a network map. 
+[Corda Network](https://corda.network/) is an publicly-available internet of Corda nodes operated by network participants. Each 
+node is identified by a certificate issued by the network's identity service, and will also be identifiable on a network map.
```
Comment (gendal) [inline docs/source/corda-network/index.md]: and will be _discoverable_ on a network map?

File: docs/source/corda-network/index.md [hunk 3]
```diff
@@ -3,57 +3,59 @@ Corda Network
...
 Corda Network enables interoperability – the exchange of data or assets via a secure, efficient internet layer – in a way 
-that isn't possible with competing permissioned distributed ledger technologies or legacy systems.
+that isn't possible with separate, isolated Corda networks. A common trust root surrounds all transactions, and a consistent set of 
+network parameters ensures all participants may transact with each other.
 
-The network is due to go live in December 2018, and initially it will be governed by R3. An independent, not-for-profit 
-Foundation is currently being set-up which is intended to govern the Network from mid 2019, after a transition period
-when control moves entirely to the Foundation. See the [governance model](governance-structure.md) for more detail.
+The network went live in December 2018, and is currently governed by R3. An independent, not-for-profit foundation has been 
+set-up to govern the network, and a transitional board of participants will be elected in Spring 2019, which will oversee the foundation
```
Comment (gendal) [inline docs/source/corda-network/index.md]: set up

Comment (gendal) [reply docs/source/corda-network/index.md]: if next year's elections are democratic does that mean this year's are NOT democratic?

File: docs/source/corda-network/index.md [hunk 4]
```diff
@@ -3,57 +3,59 @@ Corda Network
...
+The network went live in December 2018, and is currently governed by R3. An independent, not-for-profit foundation has been 
+set-up to govern the network, and a transitional board of participants will be elected in Spring 2019, which will oversee the foundation 
+until democratic elections are held a year later. See the [governance model](https://corda.network/governance/governance-guidelines.html) 
+for more detail.
 
-The Network will comprise many sub-groups many sub-groups of participants running particular CorDapps (sometimes but not 
-always referred to as 'business networks'), and such groups will often have a co-ordinating party (the 'Business 
-Network Operator') who manages the distribution of the app and rules (including around membership) for its use. 
+The network will support many sub-groups of participants running particular CorDapps (sometimes referred to as 'business networks'), 
+and these groups will often have a co-ordinating party (the 'business network operator') who manages the distribution of the
```
Comment (gendal) [inline docs/source/corda-network/index.md]: Dave H calls this role the business network governor.  Thoughts?

File: docs/source/corda-network/index.md [hunk 5]
```diff
@@ -3,57 +3,59 @@ Corda Network
...
+The Identity Service controls admissions of participants into Corda Network. The service receives certificate 
+signing requests (CSRs) from prospective network participants (sometimes via a business network operator) and reviews the 
+information submitted. A digitally signed participation certificate is returned if:
 
-* The prospective Corda Network Participant meets the requirements specified in the documentation;
-* Evidence is provided by the Participant or Business Network Operator of agreement to the Corda Network Participant Terms 
-of Use.
+* The participant meets the requirements specified in the bylaws and policies of the foundation (broadly speaking, limited to 
+sanction screening only);
+* The participant agrees to Corda Network participant terms of use.
```
Comment (gendal) [inline docs/source/corda-network/index.md]: link?

File: docs/source/corda-network/index.md [hunk 6]
```diff
@@ -3,57 +3,59 @@ Corda Network
...
-Participants, based on the Participation Certificates signed by the Doorman, and makes this information available to all 
-Corda Network Participants.
+nodes, based on the participation certificates signed by the Identity Service, and makes this information available to all 
+Corda Network nodes.
 
-Notary 
-------
+Notary Service
+--------------
 Corda design separates correctness consensus from uniqueness consensus, and the latter is provided by one or more Notary
```
Comment (gendal) [inline docs/source/corda-network/index.md]: Corda's design.  Or: The Corda design?

File: docs/source/corda-network/index.md (hunk 4 above)
Comment (jamescarlyle) [reply docs/source/corda-network/index.md]: I'm not seeing a clear distinction between the two yet in practice. In our vernacular, BNO is used much more generally than BNG. I'd prefer to keep the simplicity of a single term for this until the need to differentiate becomes clearer.

File: docs/source/corda-network/index.md (hunk 5 above)
Comment (jamescarlyle) [reply docs/source/corda-network/index.md]: y

File: docs/source/corda-network/index.md (hunk 6 above)
Comment (jamescarlyle) [reply docs/source/corda-network/index.md]: y

**Pull request merged**